commit e013c9b5a12d632f3da01fca744c0443e8a934cc
Author: Takashi Iwai <tiwai@suse.de>
Date:   Tue Sep 4 17:58:32 2018 +0200

    staging: bcm2835-audio: Clean up include files in bcm2835-ctl.c
    
    commit 821950d3da4bf97bcfedcb812176a0f26b833db0 upstream.
    
    Only a few of them are really needed.
    
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Tested-by: Stefan Wahren <stefan.wahren@i2se.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/staging/vc04_services/bcm2835-audio/bcm2835-ctl.c b/drivers/staging/vc04_services/bcm2835-audio/bcm2835-ctl.c
index 9020887e1ada..1c5a87580978 100644
--- a/drivers/staging/vc04_services/bcm2835-audio/bcm2835-ctl.c
+++ b/drivers/staging/vc04_services/bcm2835-audio/bcm2835-ctl.c
@@ -1,23 +1,8 @@
 // SPDX-License-Identifier: GPL-2.0
 /* Copyright 2011 Broadcom Corporation.  All rights reserved. */
 
-#include <linux/platform_device.h>
-#include <linux/init.h>
-#include <linux/io.h>
-#include <linux/jiffies.h>
-#include <linux/slab.h>
-#include <linux/time.h>
-#include <linux/wait.h>
-#include <linux/delay.h>
-#include <linux/moduleparam.h>
-#include <linux/sched.h>
-
 #include <sound/core.h>
 #include <sound/control.h>
-#include <sound/pcm.h>
-#include <sound/pcm_params.h>
-#include <sound/rawmidi.h>
-#include <sound/initval.h>
 #include <sound/tlv.h>
 #include <sound/asoundef.h>
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1443_e013c9b5a12d632f3da01fca744c0443e8a934cc.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1443_e013c9b5a12d632f3da01fca744c0443e8a934cc.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
